package/yast2-bootloader.changes

```diff
@@ -1,10 +1,18 @@
 -------------------------------------------------------------------
+Mon May  2 10:24:17 CEST 2016 - schubi@suse.de
+
+- Fixed nil in custom_devices. [[]].compact! is returning nil.
```

Comment:
Anyway, the changes file is user oriented so we should not fill it with implementation details (they can go to comments or commit messages). I would say
> Fixed an "undefined method reject! ... in custom_devices" (bnc#977945)